### PR TITLE
Game: add logger

### PR DESCRIPTION
**Pull request by kjarosh**

### CODE REVIEW

File: QuadradiusR/scripts/network_handler.gd [hunk 1]
```diff
@@ -32,30 +32,13 @@ func _embedded_server_ready():
 func set_url(url: String):
 	rest_api.url = url
 
-# # # # # # # #
-# CREATE USER #
-# # # # # # # #
```
Comment (Fruktus) [inline QuadradiusR/scripts/network_handler.gd:37]: Ayo, leave my section title pls

File: QuadradiusR/scripts/rest_api.gd [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class_name RestApi
 extends HTTPRequest
 
+const MODULE = 'rest'
```
Comment (Fruktus) [inline QuadradiusR/scripts/rest_api.gd:4]: this is ok but I'm pretty sure you can just print the class name

**Review by Fruktus — APPROVED** [review]
Nice idea, minor remarks, overall ok

File: QuadradiusR/scripts/network_handler.gd (hunk 1 above)
Comment (kjarosh) [reply QuadradiusR/scripts/network_handler.gd:37]: Mi scusi

**Pull request merged**